sfall/main.cpp

```diff
@@ -1454,12 +1454,13 @@ static void DllMain2() {
 	dlogr("Patching out ereg call.", DL_INIT);
 	BlockCall(0x4425E6);
 
-	tmp = GetPrivateProfileIntA("Misc", "AnimationsAtOnceLimit", 32, ini);
-	if ((signed char)tmp > 32) {
-		dlog("Applying AnimationsAtOnceLimit patch.", DL_INIT);
-		AnimationsAtOnceInit((signed char)tmp);
-		dlogr(" Done", DL_INIT);
-	}
+	AnimationsAtOnceInit();
+	//tmp = GetPrivateProfileIntA("Misc", "AnimationsAtOnceLimit", 32, ini);
```

Comment:
Never leave commented out code when working with version control, just delete it. All history is preserved.